libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -96,6 +103,27 @@ export class PickingComponent extends PickingListMixin(LitElement) {
     this.items[productIndex].status = ItemsFilters.NotPicked;
 
     this.items = [...this.items];
+
+    this.tabRef.value?.dispatchEvent(
+      new CustomEvent('click', {
+        composed: true,
+        bubbles: true,
+      })
+    );
```

Comment:
Why tab should dispatch click event?

Reply:
As I remember, we can change the tab using `activeTabIndex`.

Reply:
The `activeTabIndex` is being used mainly for the first rendering of tabs, if user change tabs manually the activeIndex won't update. Also, the tab component should dispatch an event to indicate to the component's owner that the selected property changed

Reply:
it can be ok for now, but I guess we should refactor the Tab component to make it able change tabs using prop, but not click event